src/main/resources/schema.sql

```diff
@@ -7,18 +7,18 @@ CREATE TABLE member (
 memberId BIGINT AUTO_INCREMENT,
 email VARCHAR(100) NOT NULL UNIQUE,
 password VARCHAR(100) NOT NULL,
-nickName VARCHAR(100),
-create_date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+nickname VARCHAR(100),
+create_date TIMESTAMP NOT NULL,
 PRIMARY KEY (memberId)
 );
 
 CREATE TABLE post (
 postId BIGINT NOT NULL AUTO_INCREMENT,
 title VARCHAR(255) NOT NULL,
 content TEXT NOT NULL,
-writerId  BIGINT NOT NULL,
-write_date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+writerEmail  VARCHAR(100) NOT NULL,
```

Comment:
```suggestion
writer_email  VARCHAR(100) NOT NULL,
```
보통 snake case를 mysql db에서는 많이 씁니다.

Reply:
수정하였습니다.